src/shared_modules/content_manager/src/action.hpp

```diff
@@ -67,6 +67,24 @@ class Action final
         stopActionScheduler();
     }
 
+    /**
+     * @brief Action execution with exclusivity: The action is only executed if there isn't another action in progress.
+     *
+     * @param id Action ID.
+     * @param offset Manually set current offset to process.
```

Comment:
Nitpick: I think 'manually' is unnecessary here. Maybe just 'Offset to process' ?